Client/Components/DashBoard.razor

```diff
@@ -2,6 +2,9 @@
 
 
 
+
+
+<HomeCountryDropDown @ref="dropDown" CountryChanged="HomeCountryChange" CountryNames ="@countryNames" CountryCodeDict="@countryCodeDict"/>
```

Comment:
Should this not be if the home country can't be found, where the comment says?

Reply:
Fixed